### PR TITLE
adding quotation marks to password variable

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
We need, at some point, to regenerate during upgrade the `config.json` file to keep in synch with the updated configuration changes

File: scripts/upgrade [hunk 1]
```diff
@@ -123,28 +122,11 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
...
-
-  # Remove the app directory securely
-  ynh_secure_remove --file="$final_path"
-
   if [ "$version" = "Enterprise" ]; then
-    ynh_setup_source --dest_dir="$final_path" --source_id="enterprise"
+    ynh_setup_source --dest_dir="$final_path" --source_id="enterprise" --keep="$final_path/plugins $final_path/config/config.json"
   elif [ "$version" = "Team" ]; then
-    ynh_setup_source --dest_dir="$final_path" --source_id="$architecture"
+    ynh_setup_source --dest_dir="$final_path" --source_id="$YNH_ARCH" --keep="$final_path/plugins $final_path/config/config.json"
```
Comment (kemenaran) [inline scripts/upgrade:128]: Does "--keep" merge the new content in the plugins directory with the previous one? (we want to both update built-in plugins and keep our custom plugins)

Comment (ericgaspar) [reply scripts/upgrade]: mmh, you are right, this is just preserving the plugins folder from being updated...

**Review by kemenaran — APPROVED** [review]

**Pull request merged**